3

```diff
@@ -0,0 +1,2 @@
+      invoke  active_record
```

Comment:
I believe you pushed this file by mistake. Same for `2,`. Please delete it, squash your commits and push again. You can do this by first removing the file:

```
git rm '2,' '3'
git commit -m "removing files"
```

Then rewriting your last two commits into a new one:

```
git reset --soft HEAD~2
git commit -m "cleaning up"
git push origin home_page --force
```

This will remove these files from the history.